brambling/models.py

```diff
@@ -1223,8 +1223,10 @@ def from_dwolla_refund(cls, refund, related_transaction, **kwargs):
     def can_refund(self):
         refunded = self.related_transaction_set.filter(
             transaction_type=Transaction.REFUND
-        ).aggregate(refunded=Sum('amount'))['refunded'] or 0
-        return self.amount + refunded > 0
+        ).aggregate(refunded=Sum('amount'))['refunded']
+        # None means there are no refunds, which is relevant
+        # for 0-amount transactions.
+        return refunded is None or self.amount + refunded > 0
```

Comment:
Seems a _little_ confusing for a function with a `can_` name to return a value that is neither `True` nor `False`.